### PR TITLE
Update CHANGELOG.md when exists and post a Release Notes on github

### PR DESCRIPTION
**Pull request by brunojdo**
#### What is the purpose of this pull request?

The changelog is automatically updated such as the manifest and if the user sets the env `GITHUB_API_TOKEN` we post the changes on GitHub release notes. 

#### What problem is this solving?

Now, `releasy` can automatically update projects that have a `CHANGELOG.md` and create release notes at GitHub repository. 
 
#### How should this be manually tested?
- Clone this branch
- Enter into a project which a `CHANGELOG.md`. 
- Execute `node ../releasy/bin/releasy.js`

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: lib/steps.js [hunk 1]
```diff
@@ -198,45 +221,42 @@ var steps = {
...
+        }
+        promise = promise
+            .then(function () {
+                return steps.postReleasy(config)
+            })
+            .then(function () {
+                if (!config.quiet) console.log("All steps finished successfully.");
+            });
+        promise.fail(function (reason) {
+            if (!config.quiet) console.log("Failed to release.", reason);
```
Comment (andreldsa) [inline lib/steps.js]: You can use `console.error()` with `[ERROR] Message description`

**Comment by andreldsa** [comment]
🔝 💯

File: lib/releasy.js [hunk 2]
```diff
@@ -23,6 +26,67 @@ module.exports = function(opts) {
...
+    prerelease: false,
+  }
+
+  /**
+     * Get and format GITHUB_API_TOKEN.
+     * @return Github Authorization
+    */
+  function getGithubToken() {
+    if (!process.env.GITHUB_API_TOKEN) {
+      console.log('You must to set GITHUB_API_TOKEN env if you want post the Release Notes of your project.\
```
Comment (firstdoit) [inline lib/releasy.js]: `You must to set` -> `You must set`
`if you want post` -> `if you want to post`

File: lib/releasy.js [hunk 3]
```diff
@@ -23,6 +26,67 @@ module.exports = function(opts) {
...
+    \nSee our settings section on Readme to do it.'.yellow.bold)
```
Comment (firstdoit) [inline lib/releasy.js]: `Check the settings section in README for details on how to do it`

**Pull request merged**